tests/test_center_pool.py

```diff
@@ -0,0 +1,53 @@
+"""
+CommandLine:
+    pytest tests/test_center_pool.py
+"""
+import torch
+
+from mmdet.ops import CenterPool
+
+
+def test_center_pool_device_and_dtypes_cpu():
+    """
+    CommandLine:
+        xdoctest -m tests/test_center_pool.py \
+            test_center_pool_device_and_dtypes_cpu
+    """
+    lr_tensor = torch.tensor([[[[0, 0, 0, 0, 0], [2, 1, 3, 0, 2],
+                                [5, 4, 1, 1, 6], [0, 0, 0, 0, 0],
+                                [0, 0, 0, 0, 0]]]])
+    # tb_tensor = torch.tensor([[[[0, 3, 1, 0, 0], [0, 1, 1, 0, 0],
```

Comment:
Either use or remove these commented code.
Each case need to be covered, including the assert using `pytest.raise(AssertionError)`.